### PR TITLE
MMT-1832 added language attribute to html tag.

### PR DESCRIPTION
**Pull request by rosycordova**
Please review change to <html>  to add language tag.

### CODE REVIEW

File: app/views/layouts/manage_cmr.html.erb [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!--[if IE 7]><html class="no-js lt-ie9 lt-ie8"  lang="en"> <![endif]-->
 <!--[if IE 8]><html class="no-js lt-ie9"  lang="en"> <![endif]-->
 <!--[if gt IE 8]><html class="no-js" lang="en"> <!--<![endif]-->
+<html lang="en">
```
Comment (jceaser) [inline app/views/layouts/manage_cmr.html.erb]: I see the result of the code from https://mmt.earthdata.nasa.gov/manage_cmr, so yea, this line is needed

    <!DOCTYPE html>
    
    
    
    
    <head>
      <title>Metadata Management Tool</title>

      <meta charset="utf-8">

Comment (charleshuang80) [reply app/views/layouts/manage_cmr.html.erb]: Does the `lang="en"` attribute not work in the `<!DOCTYPE html>` tag?

Comment (jceaser) [reply app/views/layouts/manage_cmr.html.erb]: I have never seen HTML 5 doc types with language, and w3c says the html team is where we do this at. https://www.w3.org/International/questions/qa-html-language-declarations

**Comment by charleshuang80** [comment]
You missed a few layout files: collection_preview.html.erb and manage_metadata.html.erb

**Comment by rosycordova** [comment]
> You missed a few layout files: collection_preview.html.erb and manage_metadata.html.erb

Thank you.  Two files updated to add language attribute to html tag.

**Pull request merged**